index.html

```diff
@@ -287,6 +291,24 @@ <h2>About Myself</h2>
           </div>
         </div>
       </section>
+
+      <!--
+        Contact section
+      -->
+      <section class="contact-form">
+        <h2>Contact me</h2>
+        
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+          I’d love to help with it!</p>
+        <form action="https://formspree.io/f/mrgvnbrj" method="POST" class="form-flex">
+          <input required type="text" class="padding-inputs" maxlength="30" name="name" id="name" placeholder="Full Name" />
+          <input required type="email" class="padding-inputs" name="email" id="email" placeholder="Email Address" />
+          <textarea rows="5" placeholder="Write your message here..." name="textarea" class="padding-inputs" id="textarea" maxlength="500" required></textarea>
+          <button type="submit">Get in touch</button>
```

Comment:
- [ ] You did great using the correct button type of `submit` here. :+1: But your button is missing the interactive attributes like the hover. I'll suggest you look at the Figma design and add the hover effect to the button. :+1: :smile:

Reply:
> [OPTIONAL] Considering that the previous code reviewer didn't address it, it would be highly beneficial to incorporate the desired functionality of the pressed button state interaction from the Figma design. This addition would enhance the user experience and align the implementation with the intended design specifications. Keep up the good work! 👍